### DISCUSSION TITLE
QoL Idea: Transfer Sessions to other device

### DISCUSSION THREAD

**Discussion by IVRYSimon** [jellyfin/jellyfin#11684] [General]
Hi guys, 

just had an awesome idea for binge watchers: Wouldn't it be super cool to transfer sessions between devices? Like in MS Teams with video and phone calls (https://support.microsoft.com/en-us/office/join-a-microsoft-teams-meeting-on-a-second-device-c28e7407-183b-46ea-ab17-2212700e5f41).

Advantages: 
- Binge watchers who would start Jellyfin on a supported client could directly jump to the exact timestamp where they left on the last device
- probably less resources are used because only one device at a time has an open stream to the jellyfin server
- smoke breaks are easier to manage (just joking)

You could add this setting as checkbox or as auto-enabled if you reached your max device limit. On the first device (session transferred away) could a small pop-up appear which indicates that the session got transferred to another device. With 2 or 3 options: Exit/Close (do nothing and close pop up), Ignore (plays the media as it would nothing happened) and "Resume". Where the "Resume" option would skip to the new timestamp of the second device where it left. 

What do you think? 

Best regards

**Comment by SplitMilky** [comment]
How is this different than opening Jellyfin on another device an continuing your show?  I have the same account for jelly on all deviced and I can stop it, get up and go to bed and continue playing.

**Reply by IVRYSimon** [reply]
@SplitMilky Yeah i got your point, but in my experience its not that easy as you said. Im skipping Outros for my shows and i got like always the wrong suggestions (all old shows show up as "continue" obviously i don´t want to see all outros :D). I usually need to search for my show and need to skip forward etc. 
Sometimes it works, most of the time not. 
Maybe a small popup (slide-in) on the bottom of the screen could also be solution. Amazon Prime has also an awkward logic - also quite frustrating. Netflix does it best imho.
This feature could also be a opt-in for "Bingewatchers" not a default on.
